src/map/lua/lua_baseentity.cpp

```diff
@@ -10378,6 +10378,24 @@ void CLuaBaseEntity::sendTractor(float xPos, float yPos, float zPos, uint8 rotat
     }
 }
 
+/************************************************************************
+ *  Function: allowSendRaisePrompt()
+ *  Purpose : Allows the raise prompt to be sent again to client
+ *            if for example player was moved while dead (thus removing the prompt)
+ *  Example : player:allowSendRaisePrompt()
+ ************************************************************************/
+
+void CLuaBaseEntity::allowSendRaisePrompt()
+{
+    XI_DEBUG_BREAK_IF(m_PBaseEntity->objtype != TYPE_PC);
```

Comment:
`XI_DEBUG_BREAK_IF` has been removed upstream. See this PR for reference: https://github.com/LandSandBoat/server/pull/3912/files

Lets add the nullptr and return a ShowError

Reply:
done